hazelcast/src/main/java/com/hazelcast/query/impl/Indexes.java

```diff
@@ -84,7 +84,7 @@ public void saveEntryIndex(QueryableEntry queryableEntry) throws QueryException
         }
     }
 
-    Index getIndex(String attribute) {
+    public Index getIndex(String attribute) {
```

Comment:
Can you add a line of javadoc (if that doesnt' exist) about the behavior? Especially what happens when the index for the attribute doesn't exist.